st2common/st2common/services/executions.py

```diff
@@ -53,7 +53,7 @@
 
 LOG = logging.getLogger(__name__)
 
-SKIPPED = ['id', 'callback', 'action', 'runner_info']
+SKIPPED = ['id', 'callback', 'action', 'runner_info', 'notify']
```

Comment:
This is actually causing an error when mongoengine is upgraded.  I tested this before and after the upgrade.  Basically, the ActionExecutionDB don't have a notify field and the code is trying to assign notify to it.  Mongoengine is complaining about it.